### DISCUSSION TITLE
3d surfaces and images

### DISCUSSION THREAD

**Discussion by DMZ0** [navis-org/navis#136] [General]
~~Nat has the class hxsurf and various functions for working with non-neuron 3d surfaces and image volumes. Is this functionality also implemented in navis? I was unable to find any info about this in the docs.~~ Edit: somehow I missed the navis.Volume class. Apologies!

**Comment by schlegelp** [comment]
No worries. Just to say that I'm happy to hear suggestions on how to make `navis.Volume` more visible if you have any.
